chef/cookbooks/cinder/recipes/volume.rb

```diff
@@ -43,6 +43,8 @@ def make_loopback_file(node, volume)
   max_fsize = ((`df -Pk #{encl_dir}`.split("\n")[1].split(" ")[3].to_i * 1024) * 0.90).to_i rescue 0
   fsize = max_fsize if fsize > max_fsize
 
+  FileUtils.mkdir_p(fdir) unless File.exists?(fdir)
```

Comment:
> What about the comment on ine 36? https://github.com/crowbar/barclamp-cinder/pull/223/files#diff-e44ebd5ea17511e644658be6d5ac87bfR36

Hm. Good point. Missed that. So I guess we should set the ownership for the newly created dir. Not allowing to create new dirs makes imho no sence. Or if we want to go this way, we should only allow to set a filename and not the whole path and default to `/var/li/cinder/` as basedir for all files!? Opinions?